### ISSUE TITLE
fsspec.gui.FileSelector, doesn't work with remote protocols? 

### ISSUE THREAD

**Issue by okz** [fsspec/filesystem_spec#1359]
FileSelector implementation suggests remote protocol support is implemented, but I haven't been able to figure out how to use it? 

Maybe the implementation has a bug with storage options? 

```
fsspec.gui.FileSelector("az://.", **so)
>> TypeError: FileSelector.__init__() got an unexpected keyword argument 'account_name'

fsspec.gui.FileSelector("file://.", storage_options=so)
>>
TypeError: FileSelector.__init__() got an unexpected keyword argument 'storage_options'

```

**Comment by martindurant** [comment]
I see the documentation on this is somewhat lacking. The GUI allows you to supply the initial value of the "kwargs" text widget, which must be a string, but evaluate to a python dict. So something like
```
fsspec.gui.FileSelector("az://.", kwargs="""{"account_name": "myname"}""")
```

**Comment by okz** [comment]
Ah thanks Martin, I understand why but definitely not obvious. 

However there is a bug after all:

The text widget is initialised with an empty dictionary, and since `storage_options` use the widget value then onwards,  an exception is raised immediately. 

Fixed the initialisation here: 
```
        --self.kwargs = pn.widgets.TextInput(name="kwargs", value="{}", align="center")
      ++self.kwargs = pn.widgets.TextInput(name="kwargs", value=self.init_kwargs, align="center")
```





storage_options, was

**Comment by martindurant** [comment]
That does seem right. If you make a PR, I will merge it. This can also be tested easily.

**Comment by ianthomas23** [comment]
This was fixed by #1360 so closing as completed.
